internal/publish/publish_test.go

```diff
@@ -108,119 +123,152 @@ func (s *PublishSuite) TestNewFromState() {
 }
 
 func (s *PublishSuite) TestPublishWithClientNewSuccess() {
-	s.publishWithClient(nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil)
+	s.publishWithClient(nil, &publishErrsMock{}, nil)
```

Comment:
Changed a bit the signature here, passing a lot of `nil`s in a certain order was growing too much

Reply:
Much better!